plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskProjectWizard.java

```diff
@@ -146,34 +128,43 @@ private void doFinish(String containerName, String fileName, String taskName, St
         IWorkspaceRoot root = ResourcesPlugin.getWorkspace().getRoot();
         IProject project = root.getProject(containerName);
         if (project.exists()) {
-            throwCoreException(HumantaskEditorConstants.THE_PROJECT_EXISTS_IN_THE_WORKSPACE_MESSAGE);
+            humanTaskWizardUtil.throwCoreException(
+                    HumantaskEditorConstants.THE_PROJECT_EXISTS_IN_THE_WORKSPACE_MESSAGE, null);
         } else {
             project.create(null);
             project.open(null);
         }
 
         IResource resource = root.findMember(new Path(containerName));
         if (!resource.exists() || !(resource instanceof IContainer)) {
-            throwCoreException("Container \"" + containerName + "\" does not exist.");
+            humanTaskWizardUtil.throwCoreException("Container \"" + containerName + "\" does not exist.", null);
```

Comment:
When we throw our own unique exception messages, without passing null for the exception object, it's better if we could create our own exception class extended from the super class to uniquely identify our module through the exception stack trace.